src/Domain/Syntax/Compound/FunctionScore.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+declare(strict_types=1);
+
+namespace JeroenG\Explorer\Domain\Syntax\Compound;
+
+use JeroenG\Explorer\Domain\Syntax\Compound\ScoreFunction\ScoreFunction;
+use JeroenG\Explorer\Domain\Syntax\MatchAll;
+use JeroenG\Explorer\Domain\Syntax\SyntaxInterface;
+
+class FunctionScore implements SyntaxInterface
+{
+    public const BOOST_MULTIPLY = 'multiply';
+
+    public const BOOST_SUM = 'sum';
+
+    public const BOOST_AVG = 'avg';
+
+    public const BOOST_FIRST = 'first';
+
+    public const BOOST_MAX = 'max';
+
+    public const BOOST_MIN = 'min';
+
+    public const BOOST_REPLACE = 'replace';
+
+    public const SCORE_MULTIPLY = 'multiply';
+
+    public const SCORE_SUM = 'sum';
+
+    public const SCORE_AVG = 'avg';
+
+    public const SCORE_FIRST = 'first';
```

Comment:
Are these Fucntion Score specific or may it be worth it to make an enum-like class?

Reply:
I believe these are FunctionScore specific, thats why I put them here